polaris/polaris/integrations/rails.py

```diff
@@ -166,5 +166,13 @@ def poll_pending_deposits(
         """
         raise NotImplementedError()
 
+    def get_indicative_price(self, buy_asset: str, sell_asset: str) -> str:
+        pass
+
+    def get_firm_price(
+        self, buy_asset: str, sell_asset: str, amount: str, expire_after: str
+    ) -> str:
+        pass
```

Comment:
I don't think these should be integration functions on `RailsIntegration` class. I don't think we need to provide an interface for Polaris to request these prices from developers in general, all we need are the functions defined on `SEP38AnchorIntegration`.